src/ln/channelmanager.rs

```diff
@@ -208,6 +208,10 @@ impl MsgHandleErrInternal {
 						},
 					}),
 				},
+				ChannelError::Stale { msg, .. } => HandleError {
+					err: msg,
+					action: Some(msgs::ErrorAction::IgnoreError),
```

Comment:
Huh? Shouldn't this return an error message to tell the peer to close on us?

Reply:
Corrected after talks on should we disclose our failures or not to our remote peer